applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -1847,7 +1848,8 @@ pub async fn command_runner(
                     } else {
                         for (i, utxo) in unblinded_utxos.iter().enumerate() {
                             println!(
-                                "{}. Value: {}, Spending Key: {:?}, Script Key: {:?}, Features: {}",
+                                "{}. Value: {}, Spending Key: {:?}, Script Key: {:?}, Features: {}, Commitment hash: \
```

Comment:
```suggestion
                                "{}. Value: {}, Spending Key: {:?}, Script Key: {:?}, Features: {}, Commitment : \
```
`utxo.1` is the commitment, not the hash